README.md

```diff
@@ -27,3 +27,78 @@ To run the cloned repository;
 [github-actions-build]: https://github.com/jhipster/jhipster-registry/workflows/Build/badge.svg
 [github-actions-docker]: https://github.com/jhipster/jhipster-registry/workflows/Docker%20Image/badge.svg
 [github-actions-url]: https://github.com/jhipster/jhipster-registry/actions
+
+## Vault Integration
```

Comment:
Maybe refer to "Hashicorp Vault" as there are several products named "Vault".

Reply:
sure. Anything else @gmarziou you would like to change?

Reply:
No, it's great